gradle-config.gradle

```diff
@@ -0,0 +1,22 @@
+//
+// Configuration file for gradle build execution.
+//
+
+task accessToken {
+  def tokenFile = new File("${projectDir}/src/main/res/values/developer-config.xml")
+  if (!tokenFile.exists()) {
+    String tokenFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
+      "<resources>\n" +
+      "    <string name=\"mapbox_access_token\">" + "$System.env.MAPBOX_ACCESS_TOKEN" + "</string>\n" +
+      "</resources>"
+
+    if (tokenFileContents == null) {
```

Comment:
@zugaldia 
> could you amend this PR with the latest version of the script? 

Sure!

> Also, instead of failing the build with an InvalidUserDataException, would it make sense instead to show a warning and default to a string like YOUR_MAPBOX_ACCESS_TOKEN_GOES_HERE?

That's [what we need to agree on](https://github.com/mapbox/mapbox-gl-native/issues/8783#issuecomment-296622518).
Before, it was implemented that way but instead of generating `YOUR_MAPBOX_ACCESS_TOKEN_GOES_HERE`, `null` was inserted.
It's a matter of _restriction_, because using new script project won't compile until you have `MAPBOX_ACCESS_TOKEN` env var set.
I imagine that this was one of the reasons why it was implemented the way as it is [here](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDKTestApp/gradle-config.gradle), so not being that hard.
That being said, what do you think?